tests/state/test_state_api.py

```diff
@@ -76,3 +79,20 @@ def test_request_cancel_without_backend(self, stub_broker, api_client):
         # if there is not cancel backend should raise an error
         assert res.json["error"] == "The default broker doesn't have a cancel backend."
         assert res.status_code == 500
+
+    def test_scheduled_jobs(self, stub_broker, api_client, do_work, scheduler, frozen_datetime):
+        timezone = pytz.timezone("Europe/Paris")
+        scheduler.schedule = [
+            ScheduledJob(
+                actor_name=do_work.actor_name, daily_time=(datetime.datetime.now(timezone)).time(), tz="Europe/Paris",
+            )
+        ]
+        scheduler.sync_config()
+
+        res = api_client.get("/scheduled/jobs")
+        jobs = res.json["result"]
+
+        for job in jobs:
```

Comment:
`        assert jobs == [{'actor_name': 'do_work', 'args': [], 'daily_time': '01:00:00', 'enabled': True, 'interval': 86400, 'iso_weekday': None, 'kwargs': {}, 'last_queued': None, 'tz': 'Europe/Paris'}]
` is simpler